enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -2368,6 +2387,13 @@ def test_can_redeem_lms_user_id_override_for_staff(
             'content_price': 19900,
         }
         self.mock_get_content_metadata.return_value = mock_subsidy_content_data
+        self.mock_catalog_get_and_cache_content_metadata.return_value = {
+            'normalized_metadata': {
+                'content_price': 199.00,  # normalized_metadata always serializes USD.
```

Comment:
nit: should this be an integer vs. float?

[Example](https://enterprise-catalog.edx.org/api/v1/content-metadata/TheEconomist+CRIT/?format=json)

```
{
  "normalized_metadata": {
    "start_date": "2025-03-10T00:00:00Z",
    "end_date": "2025-03-21T00:00:00Z",
    "enroll_by_date": "2025-03-16T00:00:00Z",
    "enroll_start_date": null,
    "content_price": 828
  },
}
```

Reply:
The serializer field type is actually a float: https://github.com/openedx/enterprise-catalog/blob/a387d9c82368405b0cda7cfa06ea96c11c0e2c04/enterprise_catalog/apps/catalog/serializers.py#L153-L154

I guess floats are serialized as seemingly integers in json when the value is close to an integer.

Reply:
Gotcha, thanks for linking to the serializer!